shell/edit/networking.k8s.io.ingress/RulePath.vue

```diff
@@ -181,48 +181,44 @@ export default {
   </div>
 </template>
 <style lang="scss" scoped>
+// TODO #11952: Correct deep statement
 $row-height: 40px;
-
-.labeled-input :deep(), :deep() .labeled-input {
-  padding: 0 !important;
-  height: 100%;
-  input.no-label {
-    height: calc($row-height - 2px);
-    padding: 10px;
-  }
-}
-.rule-path :deep() {
-  .col, INPUT {
-    height: $row-height;
-  }
-
-  .unlabeled-select {
+.rule-path {
+  :deep(.labeled-input), :deep(.labeled-input) {
```

Comment:
I'm curious as to why we need the double `:deep()` selectors 🤔

Something tells me that we leave this as is for now and we can circle back to better understand the intent in the future..

Reply:
Maybe it had a reason before, but not now 😅